docs/main/faqs/cStor.md

```diff
@@ -0,0 +1,328 @@
+---
+id: cStorFAQ
+title: cStor FAQs
+keywords: 
+ - cStor FAQ
```

Comment:
I believe it will be better to have few more keywords here.
- rancher prerequisites
- pool creation
- rancherOS